Client/Frontend/Browser/UserScriptManager.swift

```diff
@@ -174,16 +174,29 @@ class UserScriptManager {
   /// The first part is handled by an ad-block rule and enabled via a `deAmpEnabled` boolean in `AdBlockStats`
   /// The third part is handled by debouncing amp links and handled by debouncing logic in `DebouncingResourceDownloader`
   private let deAMPUserScript: WKUserScript? = {
-    guard let path = Bundle.current.path(forResource: "DeAMP", ofType: "js"), let source: String = try? String(contentsOfFile: path) else {
+    guard let path = Bundle.current.path(forResource: "DeAMP", ofType: "js"), var source: String = try? String(contentsOfFile: path) else {
       log.error("Failed to load cookie control user script")
       return nil
     }
-
-    return WKUserScript.create(
-      source: source,
-      injectionTime: .atDocumentStart,
-      forMainFrameOnly: true,
-      in: .defaultClient)
+    
+    do {
+      let arguments = try UserScriptHelper.makeDeAmpScriptParamters()
+      
+      source = [
+        source,
+        "window.braveDeAmp(\(arguments))",
+        "delete window.braveDeAmp"
+      ].joined(separator: "\n")
```

Comment:
nit: Would be a bit more clear to use:
```swift
"""
\(source)
window.braveDeAmp(\(arguments));
delete window.braveDeAmp;
"""
```

Reply:
Honestly I don't like that way that `"""` formats things (Because it includes your tabbing) plus I think joined is slightly faster than `"\(somestring)"`